doc/NOTICE

```diff
@@ -56,25 +56,25 @@ Apache Forrest includes the following in its NOTICE file:
 | http://www.krysalis.org/
 |
 | This product includes software developed by Andy Clark.
-| http://people.apache.org/~andyc/neko/
+| https://people.apache.org/~andyc/neko/
 |
 | This product includes software developed by the ExoLab Project
-| http://www.exolab.org/
+| https://www.exolab.org/
 |
 | This product includes software developed by TouchGraph LLC
-| http://www.touchgraph.com/
+| https://www.touchgraph.com/
 |
 | This product includes software developed by Marc De Scheemaecker
 | http://nanoxml.cyberelf.be/
```

Comment:
You've missed some of these as well. But I don't think al lot of these mentioned packages are still being used. For example, I don't see any direct evidence of using nanoxml or antrl.

Reply:
Fixed.
Perhaps we should to a cleanup after this has been completed.

Reply:
Yes, I've created a ticket to pick this up later: https://jira.apache.org/jira/browse/AVRO-2435